tests/test_cli.py

```diff
@@ -208,3 +242,134 @@ def mock_list(this, details=False, new=False):
     result = runner.invoke(cli, ["list", "-b", "ldp", "--ldp-endpoint", "ovh-eu"])
     assert result.exit_code == 0
     assert "Configured ldp backend contains no archive" in result.output
+
+
+def test_list_command_with_fs_backend(monkeypatch):
+    """Test the list command using the LDP backend"""
+
+    archive_list = [
+        "file1",
+        "file2",
+    ]
+    archive_list_details = [
+        {
+            "filename": "file1",
+            "modifiedAt": "1604661859",
```

Comment:
I prefer using snake case (see my previous comment):

```suggestion
            "modified_at": "1604661859",
```

Reply:
Fixed